tests/pthread_rwlock/Makefile

```diff
@@ -12,7 +12,7 @@ DISABLE_MODULE += auto_init
 
 CFLAGS += -DNATIVE_AUTO_EXIT
 
-BOARD_INSUFFICIENT_RAM += chronos mbed_lpc1768 msb-430 msb-430h stm32f0discovery \
+BOARD_INSUFFICIENT_MEMORY += chronos mbed_lpc1768 msb-430 msb-430h stm32f0discovery \
```

Comment:
It's unrelated to this PR but why is it `+=` here and `:=` in the other applications?